src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -0,0 +1,191 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class VizShonenJump extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'vizshonenjump';
+        super.label = 'Viz - Shonen Jump';
+        this.tags = [ 'manga', 'english' ];
+        this.url = 'https://www.viz.com';
+        this.requestOptions.headers.set( 'x-referer', this.url );
+        this.config = {
+            throttle: {
+                label: 'Throttle Requests [ms]',
+                description: 'Enter the timespan in [ms] to delay consecuitive HTTP requests.\nThe website may ban your IP for to many consecuitive requests.',
```

Comment:
Its more something like `The image links may expire for too many consecutive requests and therefore the downloads may fail.`